### PR TITLE
Add examples of using postFilterThreshold and remove query tensor type declaration files

### PR DESCRIPTION
**Pull request by jobergum**
…take one

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jobergum** [comment]
If you don't mind taking a pass @geirst I would appreciate it.

File: en/nearest-neighbor-search.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+}
+rank-profile hamming-nn {
+    num-threads-per-search: 12
+    first-phase {
+        expression: closeness(field,vector)
+    }
+}
+</pre>
+
+Hamming distance search over binary vectors is implemented with xor and pop count cpu instructions,
```
Comment (geirst) [inline en/nearest-neighbor-search.md:366]:
```suggestion
Hamming distance search over binary vectors is implemented with xor and pop count cpu instructions.
```

**Review by geirst — APPROVED** [review]
👍  This is great! Some minor comments to look at.

File: en/approximate-nn-hnsw.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+that exact searches over large vector volume require adjustment of the 
+[query timeout](reference/query-api-reference.html#timeout). Default Vespa query timeout is 500ms, which will
+be too low for an exact search over many vectors. 
+
+In addition to [targetHits](reference/query-language-reference.html#targethits), 
+there is a [hnsw.exploreAdditionalHits](reference/query-language-reference.html#hnsw-exploreadditionalhits) parameter
+which controls how many extra nodes in the graph (in addition to `targetHits`)
+that are explored during the graph search. This parameter is used to tune accuracy quality versus query performance. 
+
+## Combining approximate nearest neighbor search with filters
```
[the remark this answers is not in the document]
Comment (jobergum) [reply en/approximate-nn-hnsw.md:126]: Was thinking to link to the blog post, but mention the parameters?

[the remark this answers is not in the document]
Comment (geirst) [reply en/approximate-nn-hnsw.md:126]: Yes, that sounds good.

[the remark this answers is not in the document]
Comment (jobergum) [reply en/approximate-nn-hnsw.md:126]: Thanks

**Pull request merged**